tox.ini

```diff
@@ -48,7 +48,7 @@ deps =
     ruff
 
 commands =
-    ruff request_token
+    ruff check request_token
```

Comment:
Just curious, what is this for?

Reply:
Ruff changed the command we need to do at some point in the past in few months.

Reply:
See [here](https://github.com/yunojuno/django-request-token/actions/runs/11478467966/job/31942741668). It needs the `check` command.